CHANGELOG.md

```diff
@@ -7,21 +7,35 @@
 [Full log](https://github.com/sofa-framework/sofa/compare/v20.06...HEAD)
 
 
-### Deprecated
+### Breaking
 
+- Ongoing [SOFA-NG project](https://github.com/sofa-framework/sofa/issues/1527) generate several breaking changes due to modularization. It currently targets SOFA packages (SofaMisc, [SofaGeneral](https://github.com/sofa-framework/sofa/issues/1529) and soon SofaCommon), as well as SofaKernel (SofaBase\*, SofaCore, SofaHelper, SofaSimulation). Follow the [progress of the project online](https://github.com/sofa-framework/sofa/projects/9#card-47075575).
+  - [Modularization] SofaMisc* (Revival) [#1520](https://github.com/sofa-framework/sofa/pull/1520)
```

Comment:
I would put a line before like: 
Any cmake configuration error could come from PR Y
and before the size_t change: any compilation error link to type missmatch or unknow signature method could come from PR X